sdk/storage/Azure.Storage.Blobs/README.md

```diff
@@ -127,11 +123,7 @@ We fully support both synchronous and asynchronous APIs.
 string downloadPath = "hello.jpg";
 
 // Download the public blob at https://aka.ms/bloburl
-BlobDownloadInfo download = await new BlobClient(new Uri("https://aka.ms/bloburl")).DownloadAsync();
-using (FileStream file = File.OpenWrite(downloadPath))
-{
-    await download.Content.CopyToAsync(file);
-}
+await new BlobClient(new Uri("https://aka.ms/bloburl")).DownloadToAsync(downloadPath);
```

Comment:
Wait - why is this here twice?  Did we get the snippets wrong?  Should this be the enumerating blobs sample?

Reply:
This is the Async snippet

Reply:
Ah, sorry.  The diff said it was part of enumeration.  My bad.